lib/blacklist.js

```diff
@@ -1,4 +1,5 @@
 const db = require('./db')
+const notifier = require('./notifier/index')
```

Comment:
I believe that, if you omit the `/index`, it knows that's the entrypoint by default

```suggestion
const notifier = require('./notifier')
```